web/html/src/components/input/Check.js

```diff
@@ -1,18 +1,20 @@
 // @flow
 
-const React = require('react');
-const { InputBase } = require('./InputBase');
+import React from 'react';
+import { InputBase } from './InputBase';
+import { FormContext } from './Form';
 
 type Props = {
   inputClass?: string,
 } & InputBase.Props;
 
-function Check(props: Props) {
+export function Check(props: Props) {
   const {
     label,
     inputClass,
     ...propsToPass
   } = props;
+  const context = React.useContext(FormContext);
```

Comment:
You can consider naming this `formContext` to be more specific (in other components as well).

Reply:
:heavy_check_mark: